src/views/Gallery.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { FunctionComponent, useEffect, useState } from 'react';
+import { navigate, RouteComponentProps } from '@reach/router';
+import styled from 'styled-components';
+import useSWR from 'swr';
+import Skeleton from 'react-loading-skeleton';
+
+import { Alert, Button, Heading, Modal } from '../components';
+import useSessionStorage from '../hooks/useSessionStorage';
+
+type GalleryProps = {} & RouteComponentProps;
+
+type ImageType = {
+  id: number;
+  thumbnailUrl: string;
+  title: string;
+  url: string;
+};
+
+const HeadingWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: ${(props) => props.theme.sizes.baseSpacer};
+`;
+
+const User = styled.span`
+  margin-right: ${(props) => props.theme.sizes.halfSpacer};
+`;
+
+const GalleryWrapper = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+  grid-gap: ${(props) => props.theme.sizes.baseSpacer};
+`;
+
+const StyledThumbnail = styled.img`
+  cursor: pointer;
+  width: 100%;
+  transition: all 0.3s ease-in-out;
+  &:hover,
+  &:focus {
+    transform: scale(1.05);
+  }
+`;
+
+const StyledFigure = styled.figure`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+`;
+
+const Gallery: FunctionComponent<GalleryProps> = () => {
+  const { session, clear } = useSessionStorage('tartanHub');
+  const { data, isValidating, error } = useSWR(
+    'https://jsonplaceholder.typicode.com/albums/1/photos'
+  );
```

Comment:
```suggestion
  const { data, isValidating, error } = useSWR<ImageType[]>(
    'https://jsonplaceholder.typicode.com/albums/1/photos'
  );
```

You can flow the expected response type through to `data` via the first generic type arg.